tests/test_pydantic_json_response_class.py

```diff
@@ -0,0 +1,39 @@
+import math
+from datetime import date
+from typing import List
+
+from fastapi import FastAPI
+from fastapi.responses import PydanticJSONResponse
+from fastapi.testclient import TestClient
+from pydantic import BaseModel
+
+from .utils import needs_pydanticv2
+
+app = FastAPI(default_response_class=PydanticJSONResponse)
+
+
+class Response(BaseModel):
+    today: date
+    numbers: List[float]
+
+
+@app.get("/basic_response")
+def get_basic_response() -> Response:
+    return Response(
```

Comment:
```suggestion
def get_basic_response() -> ResponseModel:
    return ResponseModel(
```

`Response` is very confusing, it took me some time to notice that this is not `Response` imported from `fastapi` but user-defined model